### DISCUSSION TITLE
How is this different from redaxios ?

### DISCUSSION THREAD

**Discussion by uchihamalolan** [sindresorhus/ky#359] [Q&A] [answered]
I was searching for alternatives to axios, since I required only a basic fetch wrapper. Two libraries were interesting. One is `ky` and the other is [redaxios](https://github.com/developit/redaxios)

Redaxios offers a familiar `axios` syntax (kinda like how day.js is for moment)

I wanted to know, is there any significant reasons to prefer `ky` over `redaxios`?
(Since I find 2 reasons to use redaxios (i.e. familiar syntax & lesser size))

**Comment by szmarczak** [comment, marked as answer]
`ky` supports timeouts, retries and hooks. `redaxios` does not.

**Comment by sholladay** [comment]
Ky has solutions for nearly every open issue I see on redaxios. We have hooks which are like interceptors, we support aborting requests, we support timeouts, we support Deno, we normalize the case of request methods, we support custom config defaults, and I believe https://github.com/developit/redaxios/issues/65 wouldn't happen with Ky.
